README.md

```diff
@@ -1,24 +1,119 @@
-# README
+# BOOK_APPOINTMENT-BACK-END
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+This is backend API for the **Doctor Appointment-Front-End** UI, built with Ruby-on-Rails and Postgresql. This API manages reservations for patient and different specialize doctors.
 
-Things you may want to cover:
 
-* Ruby version
+## Built with
 
-* System dependencies
+- Ruby version 3.1.2
+- Rails (RoR) version 7.0.4
+- Postgresql
+- Rubocop
 
-* Configuration
+## Live Link
 
-* Database creation
+You can access the Live version and use the API via [this link](https:).
 
-* Database initialization
+## Back-End
 
-* How to run the test suite
+You can access the front-end of this application via [this link]https://github.com/efoosag/appointment-back-end.git).
 
-* Services (job queues, cache servers, search engines, etc.)
+## Kanban Board:
+Kanban Board [link](https://github.com/users/efoosag/projects/9)
+**Initial state of the Kanban Board:**
+![final capstone kanban board](https://user-images.githubusercontent.com/96612103/202553426-d53bf33f-bc93-4a88-bf2f-124899c42bac.png)
 
-* Deployment instructions
 
-* ...
+## SetUp
+
+To get a local copy up and running, follow these steps:
+
+- You should have **_Ruby_** installed on your computer.
+- You should also have **_Rails_** installed on your computer.
+- Clone this repository into your computer by running the following command in your terminal:
+
+```bash
+> git clone https://github.com/efoosag/appointment-back-end.git
+> cd book-appointment-back-end
+```
+
+- Run this command in the cloned directory, to install required gems:
+
+```
+> bundle install
+```
+
+- Then create the posgreSQL database needed for this app, run the migrations and seed data with the following commands:
+
+```bash
+> rails db:create
+> rails db:migrate
+> rails db:seed
+```
+
+- Run `Rubocop -A` and fix linters errors, if any.
+
+* Run the project in your browser:
+
+```
+rails server
+```
+
+## Run Tests
+
+- To run tests, Run the following command on your terminal
+
+```
+rake rswag
+```
```

Comment:
- [ ] Your readme tasks users to run tests, but I couldn't find any.  I also ran the instructed command and didn't find any.  Kindly add tests to ensure your app's integrity. 💪